agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -533,6 +543,13 @@ private static SpanExporter createSpanExporter(
                 return true;
               }
               return false;
+            },
+            (span, event) -> {
+              Double samplingPercentage =
+                  new SamplingOverrides(exceptionSamplingOverrides)
```

Comment:
you can move the `new SamplingOverrides()` construction outside of the lambda (for better perf, only need to create it once)